docs/NOTICES.md

```diff
@@ -734,6 +735,19 @@
 ##### Affected files
 [All GTFS files supported by the specification.](http://gtfs.org/reference/static#dataset-files)
 
+#### [LocationTypeStationForStopOnPathwayNotice](/RULES.md#LocationTypeStationForStopOnPathwayNotice)
+##### Fields description
+
+| Field name   	| Description                   	| Type   	|
+|--------------	|-------------------------------	|--------	|
+| `pathwayId`   	| The id of the faulty row from `pathways.txt`. 	| String 	|
+| `pathwayCsvRowNumber`| The row of the faulty record. 	| Long   	|
+| `stopId`  	| The id of the faulty row from `stops.txt`.   	| String 	|
+| `stopCsvRowNumber` 	| Faulty value.                 	| String 	|
```

Comment:
I would add the field name effected in pathways since it can be either `from_stop_id` or `to_stop_id`